### ISSUE TITLE
Accessibility issues - first rough list

### ISSUE THREAD

**Issue by patrickhlauke** [sweetalert2/sweetalert2#564]
Dumping my unsorted list of observations in here. These may likely need to be split into separate discrete issues...

- [x] "Custom HTML..." - thumbs-down button example lacks label/name; focus set to "great", user has to reverse-tab to get to links. best to set focus to start of content instead
- [x] "A warning message, with a function attached to the "Confirm"-button..." the second modal (after "Yes delete it") is not announced, only "OK" button is
- [x] ditto for following "... and by passing a parameter, you can execute something else for "Cancel"."
- [x] "A message with a custom image and CSS animation disabled" image lacks proper alt
- [x] "Ajax request example" input lacks label/name. visible custom error message not associated with input. busy indicator not announced. temporarily hear focus moving to top of page before new modal appears
- [x] "Dynamic queue" temporarily hear focus moving to top of page. busy indicator not announced. result modal still has "Your public IP will be received via AJAX request" text somewhere
- [x] Inputs/form controls lack labels (may need a new parameter to be able to provide explicit label?):  input types example, email example, password example, textarea example, select example, range example, multiple inputs example
- [x] select: choosing not ukraine shows error message, but this is not announced/associated at all; selecting ukraine shows success modal which is not announced, only "OK button"
- [x] radio: busy indicator at start not announced; success/result modal not announced
- [x] checkbox: error (when continuing unchecked) not announced/associated; success modal not announced
- [x] multiple inputs: success modal not announced

Improvement

- [x] non-alert part not `aria-hidden` / `inert` (e.g. using JAWS, can use cursor key up to navigate back out into page) - may require extensive work though
- [x] in demo page itself: clearer contrast and focus indication

**Comment by acSpock** [comment]
htmlTemplateUrl instead of inlining html... would be nice too

**Comment by limonte** [comment]
Hi @patrickhlauke !

I just checked two items in your list as done. 

I would like you to confirm if the biggest issue "next modal isn't announced" still here. I tested with ChromeVox on Linux, NVDA on Windows and VoiceOver on iPhone and everything works fine. 
Maybe that's a JAWS-related issue? If yes, what do you think about adding `aria-live="assertive"` to the modal?

**Comment by patrickhlauke** [comment]
sorry, after filing this i then completely forgot to follow-up and check in/reply. apologies, didn't want to be so rude, but things got in the way (_waves at everything_). i do appreciate that you took the time to address these 🚀
